README.md

```diff
@@ -12,12 +12,24 @@ Sagat is a full-stack application for managing IOTA blockchain multisig wallets,
 ## Starting to work locally
 
 First create a `.env` file `/api/.env` based on the `.env.example` file.
+
+```sh
+cp api/.env.test api/.env
+
+```
+
 Start postgres locally using Docker:
 
 ```sh
 docker compose up postgres -d
 ```
 
+The first time run the migrations to create the database schema:
+
+```sh
+(cd api && bun run db:migrate)
```

Comment:
Before this you need to do `bun install` which is the next step, so should probably do this between `bun install` and `bun run dev`

Reply:
https://github.com/iotaledger/sagat/pull/17/commits/f1c2bb2225953679bd5cd329e8aeca373078e4db